content/applications/inventory_and_mrp/quality/quality_check_types/picture_check.rst

```diff
@@ -2,6 +2,7 @@
 Take a Picture quality check
 ============================
 
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
```

Comment:
I can't add a comment on the exact line, but on line 24, the link for "quality control points" goes to the "quality checks" doc when it should go to the QCP doc :)

Also, on line 43, the heading should be in sentence case (i.e. "Quality control point (QCP)")